src/test/java/com/icthh/xm/ms/entity/web/rest/XmEntityResourceExtendedIntTest.java

```diff
@@ -1886,109 +1559,6 @@ public void doubleSaveUniqueField() {
 
     }
 
-    @Test
-    @Transactional
-    public void manageXmEntityAvatarUrl() throws Exception {
```

Comment:
@osiievska23 why this test was removed?

Reply:
This test involves elasticsearch usage and was moved to XmEntityResourceExtendedElasticsearchTest.class